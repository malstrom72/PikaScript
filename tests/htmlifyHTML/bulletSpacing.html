
<ul>
<li>Apple<br>tastes good
<br><br>indeed
<br><br></li>
<li>Banana<br>is tasty
<br><br>yes indeed
<br><br></li>
<li>Carrot<br>is crunchy
<br><br>yes
</li>
</ul>