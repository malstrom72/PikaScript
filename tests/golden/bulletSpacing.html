
<ul>
<li>Apple<br>tastes good
<br><br>indeed
</li>
</ul>
